gulpfile.js

```diff
@@ -18,7 +18,7 @@ const nls = require('vscode-nls-dev');
 const tsProject = ts.createProject('./tsconfig.json', { typescript });
 
 const inlineMap = true;
-const inlineSource = false;
+const inlineSource = true;
```

Comment:
Not being familiar with gulp what is the effect of this switch?.

Reply:
I was debugging, this is a mistake and can be put back as well :)